.docker/public-nginx.conf

```diff
@@ -39,6 +39,22 @@ server {
     listen 80;
     root /app/browser/public;
 
+    listen 443 ssl;
+
+    ssl_certificate     /etc/nginx/https_certificate.pem;
```

Comment:
these files won't be present locally, so I'll put some dummies in `docker-compose.override.yml`. Normally HTTPS doesn't work locally i.e. it works only with a self-signed certificate and `curl -k`.

Reply:
The whole configuration should be extracted into a `/etc/nginx/https.conf` file.

Reply:
Better yet, that means we can put an dummy `https.conf` file rather than self-signed broken certificates.